src/Components/App/App.js

```diff
@@ -1,5 +1,6 @@
 import React, { Component } from 'react';
 import './App.css';
+import { Route, NavLink } from 'react-router-dom';
```

Comment:
We can remove NavLink since it doesn't look like we will be using that under App.js